src/drive/mobile/containers/DriveMobileRouter.jsx

```diff
@@ -11,9 +11,10 @@ import { restoreCozyClientJs, initBar } from 'drive/mobile/lib/cozy-helper'
 class DriveMobileRouter extends Component {
   afterAuthentication = async ({ url, clientInfo, token, router }) => {
     const wasRevoked = this.props.isRevoked
+    this.context.client.options.uri = url
+    await restoreCozyClientJs(url, clientInfo, token)
```

Comment:
This needs to be done *before* calling `saveCredentials`, because that ends up starting a replication which needs a valid cozy-client-js.